client-side/src/stories/DateTime/DateInputs.scss

```diff
@@ -0,0 +1,24 @@
+@import '../stylies.scss';
+
+.timeInputWrapper .MuiOutlinedInput-root .MuiOutlinedInput-notchedOutline {
```

Comment:
please use scss hierarchic,  the code that you wrote is CSS, not scss.
for example:
 .timeInputWrapper .MuiOutlinedInput-root{
    .MuiOutlinedInput-notchedOutline {
        border-color: $secondary-color;
    }
    &.Mui-focused .MuiOutlinedInput-notchedOutline {
        border-color: $secondary-color;
    }
}